### PR TITLE
Backport department account icon to 2.5 & Maven wrapper

### PR DESCRIPTION
**Pull request by byilagan808**
# Ticket Link

[TICKET:1704](https://uhawaii.atlassian.net/browse/GROUPINGS-1704)

# List of squashed commits

- Backport Maven wrapper
- Backport department account icon
- Trashcan icon in its own column

# Test Checklist

- [x] Exhibits Clear Code Structure:
- [x] Project Unit Tests Passed:
- [ ] Project Jasmine Tests Passed:
- [x] Executes Expected Functionality:
- [x] Tested For Incorrect/Unexpected Inputs:

### CODE REVIEW

File: src/main/resources/templates/fragments/exclude.html [hunk 1]
```diff
@@ -18,7 +19,7 @@ <h1 class="font-weight-bold text-dark pt-2 mb-0 d-inline-block">Exclude ({{group
             </div>
         </div>
 
-        <div class="table-responsive-sm">
```
Comment (JorWo) [inline src/main/resources/templates/fragments/exclude.html:21]: This should stay as a div.

File: src/main/resources/templates/fragments/include.html [hunk 2]
```diff
@@ -18,7 +19,7 @@ <h1 class="font-weight-bold text-dark pt-2 mb-0 d-inline-block">Include ({{group
             </div>
         </div>
 
-        <div class="table-responsive-sm">
```
Comment (JorWo) [inline src/main/resources/templates/fragments/include.html:21]: This should stay as div.

File: src/main/resources/templates/fragments/exclude.html [hunk 3]
```diff
@@ -18,7 +19,7 @@ <h1 class="font-weight-bold text-dark pt-2 mb-0 d-inline-block">Exclude ({{group
...
+        <span class="table-responsive-sm">
```
Comment (JorWo) [inline src/main/resources/templates/fragments/exclude.html:22]: This should stay as a div.

File: src/main/resources/templates/fragments/include.html [hunk 4]
```diff
@@ -18,7 +19,7 @@ <h1 class="font-weight-bold text-dark pt-2 mb-0 d-inline-block">Include ({{group
...
+        <span class="table-responsive-sm">
```
Comment (JorWo) [inline src/main/resources/templates/fragments/include.html:22]: This should stay as a div.

File: src/main/resources/templates/fragments/exclude.html (hunk 3 above)
Comment (byilagan808) [reply src/main/resources/templates/fragments/exclude.html]: Added another commit to branch resolving this issue.

File: src/main/resources/templates/fragments/include.html (hunk 4 above)
Comment (byilagan808) [reply src/main/resources/templates/fragments/include.html]: Added another commit to branch resolving this issue.

File: src/main/resources/templates/fragments/owners.html [hunk 5]
```diff
@@ -18,16 +18,17 @@ <h1 class="font-weight-bold text-dark pt-2 mb-0 d-inline-block">Owners ({{groupi
             </div>
         </div>
 
-        <div class="table-responsive">
+        <span class="table-responsive">
```
[the remark this answers is not in the document]
Comment (byilagan808) [reply src/main/resources/templates/fragments/owners.html]: Added another commit to branch resolving this issue.

**Review by JorWo — APPROVED** [review]
Thanks for backporting!

**Pull request merged**